azure-pipelines-docs-integration.yml

```diff
@@ -94,6 +95,16 @@ stages:
           inputs:
               secureFile: 'aws_config'
               retryCount: '2'
+
+        - script: |
+            curl -sS https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-367.0.0-linux-x86_64.tar.gz > ./google-cloud-sdk-367.0.0-linux-x86_64.tar.gz && tar -xf ./google-cloud-sdk-367.0.0-linux-x86_64.tar.gz
+            ./google-cloud-sdk/install.sh --usage-reporting=False --path-update=True --quiet --install-python=False
+            ./google-cloud-sdk/bin/gcloud config configurations create superconductive-internal-configurations
+            ./google-cloud-sdk/bin/gcloud config set account sandbox-account-for-azure-tests --project=superconductive-internal --access-token-file=$(gcp_authkey.secureFilePath)
+            ./google-cloud-sdk/bin/gcloud config set pass_credentials_to_gsutil True
+            ./google-cloud-sdk/bin/gcloud auth activate-service-account --key-file=$(gcp_authkey.secureFilePath)
+          displayName: 'Install Google Cloud SDK'
```

Comment:
Google Cloud SDK is required to test `gsutil` and could be leveraged in future docs that demonstrate `gcloud` commands.

Reply:
amazing. Thank you for including this 🙇🏼

Reply:
❤️